int32            field1 # 40
MigratedExplicit field2 # (17, 58.2, "aldfkja", 82)
string           field3 # "radasdk"
